libraries/AP_OSD/AP_OSD.h

```diff
@@ -20,14 +20,50 @@
 #include <AP_Param/AP_Param.h>
 #include <AP_Math/AP_Math.h>
 #include <AP_BLHeli/AP_BLHeli.h>
+#include <RC_Channel/RC_Channel.h>
+#include <AP_Param/AP_Param.h>
 
 #ifndef OSD_ENABLED
 #define OSD_ENABLED 0
 #endif
 
 class AP_OSD_Backend;
 
-#define AP_OSD_NUM_SCREENS 4
+#define AP_OSD_NUM_DISPLAY_SCREENS 4
+#define AP_OSD_NUM_PARAM_SCREENS 2
+#define AP_OSD_NUM_SCREENS (AP_OSD_NUM_DISPLAY_SCREENS + AP_OSD_NUM_PARAM_SCREENS)
+
+#define SYM_M           0xB9
```

Comment:
All of these defines are going to be present in everything which includes `AP_OSD.h`.  Could we look at moving these into the implementation, or as static-consts in the object?

Reply:
moved into implementation